templates/docs/examples/layouts/application/JAAS.html

```diff
@@ -0,0 +1,565 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Application / JAAS{% endblock %}
+
+{% block style %}
+<style>
+  {% include "docs/examples/layouts/application/_styles.css" %}
+
+  @media (min-width: 772px) {
+    .l-aside.is-jaas {
+      max-width: calc(100vw - 3rem - 20rem);
+    }
+  }
+
+  @media (min-width: 1036px) {
+    .l-aside.is-jaas {
+      max-width: calc(100vw - 15rem - 20rem);
+    }
+  }
+</style>
+{% endblock %}
+
+{% block content %}
+<div class="l-application" role="presentation">
+  <div class="l-navigation-bar">
+    <div class="p-panel is-dark">
+      <div class="u-fixed-width u-align--right">
+        <span class="js-menu-toggle">Menu</span>
+      </div>
+    </div>
+  </div>
+
+  <header class="l-navigation is-collapsed">
+    <div class="l-navigation__drawer">
+      <div class="p-panel is-dark">
+        <div class="p-panel__header--sticky">
+          <div class="u-fixed-width u-sv-2">
+            <div class="p-inline-list--stretch">
+              <div class="p-inline-list__item">
+                <a class="p-panel__logo" href="#">
+                  <img src="https://assets.ubuntu.com/v1/0b2b7bce-Vanilla-logo_88x24px_white.svg" alt="Vanilla framework" height="24">
```

Comment:
As a JAAS example, it would be nice to have the JAAS logo.

Reply:
We will be updating logo and side navigation separately, so we can use the JAAS logo then.

Reply:
Ok, I noticed some issues with the side nav which I assumed was a follow-up. Though as this was just a logo would be ok but either works.